admin/javascript/src/components/form/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+import FormActionComponent from 'components/form-action/index';
+
+class FormComponent extends SilverStripeComponent {
+
+  /**
+   * Gets the components responsible for perfoming actions on the form.
+   * For example form submission.
+   *
+   * @return array|null
+   */
+  getFormActionComponents() {
+    return this.props.actions.map((action) =>
+      <FormActionComponent {...action} />
+    );
+  }
+
+  render() {
+    const attr = this.props.attributes;
```

Comment:
That's my bad. I missed the next line where this is used as a spread in `<form {...attr}>` ... or maybe that's new, but either way, not needed anymore.